packages/static-module-record/src/transform-analyze.js

```diff
@@ -65,6 +64,8 @@ const makeCreateStaticRecord = transformSource =>
       hoistedDecls: [],
       importSources: Object.create(null),
       importDecls: [],
+      // enables passing import.meta usage hints up.
+      importMeta: { uttered: false },
```

Comment:
I don't like the looks of it, but how else do I pass a boolean by reference? 
At least it's future proof if we want more information out of it.

Reply:
Passing an `importMetaSeen()` function instead might be clearer.